README.md

```diff
@@ -1,13 +1,33 @@
-# Project Todos
+# Project Todos - https://sammys-todo.netlify.app/
```

Comment:
I really like your ReadMe!! Def going to take after you and add a bit more fun to my ReadMe's in the future